nixos/modules/programs/openvpn3.nix

```diff
@@ -1,43 +1,116 @@
 { config, lib, pkgs, ... }:
-
 let
+  json = pkgs.formats.json { };
   cfg = config.programs.openvpn3;
```

Comment:
```suggestion
  cfg = config.programs.openvpn3;

  inherit (lib) mkEnableOption mkPackageOption mkOption literalExpression max options lists;
  inherit (lib.types) bool submodule ints;
```

Reply:
Given that all of this imports are specific to building module options, is it worth importing them for the whole module?

Reply:
I think using imports for the entire module is more readable, because you don't use nested let statements. It's also easier to understand - I mean, you don't usually scope the stdlib to module options, so choosing to do that raises some questions.

Reply:
:white_check_mark: Moved the imports to the top-level declaration

Reply:
Done :white_check_mark: